doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -770,11 +770,6 @@ Bluetooth libraries and services
 
   * Added experimental support for a new cryptographical backend that relies on the PSA crypto APIs (:kconfig:option:`CONFIG_BT_FAST_PAIR_CRYPTO_PSA`).
 
-Bootloader libraries
```

Comment:
Removed the section for the bootloader libraries as they are moved under Security libraries section.